Formula/proftpd.rb

```diff
@@ -1,11 +1,12 @@
 class Proftpd < Formula
   desc "Highly configurable GPL-licensed FTP server software"
   homepage "http://www.proftpd.org/"
-  url "https://github.com/proftpd/proftpd/archive/v1.3.7a.tar.gz"
-  mirror "https://fossies.org/linux/misc/proftpd-1.3.7a.tar.gz"
-  mirror "https://ftp.osuosl.org/pub/blfs/conglomeration/proftpd/proftpd-1.3.7a.tar.gz"
-  sha256 "8b7bbf9757988935352d9dec5ebf96b6a1e6b63a6cdac2e93202ac6c42c4cd96"
-  license "GPL-2.0"
+  url "https://github.com/proftpd/proftpd/archive/v1.3.7b.tar.gz"
+  mirror "https://fossies.org/linux/misc/proftpd-1.3.7b.tar.gz"
+  mirror "https://ftp.osuosl.org/pub/blfs/conglomeration/proftpd/proftpd-1.3.7b.tar.gz"
+  version "1.3.7b"
```

Comment:
Confirmed that `version` is necessary, otherwise it is parsed without the incrementing letter.